src/Microsoft.Data.SqlClient/tests/tools/Microsoft.Data.SqlClient.TestUtilities/Config.cs

```diff
@@ -42,6 +42,7 @@ public class Config
         public string KerberosDomainUser = null;
         public bool IsManagedInstance = false;
         public string AliasName = null;
+        public bool IsJsonSupported = false;
```

Comment:
Is this something you can detect after making connection?

Reply:
@cheenamalhotra 
Can't be done without turning on Feature Switches on Debug builds of Sql server today. 
We will need a hardcoded config value at this moment unfortunately :( 

also question for my education @cheenamalhotra . In ConditionalFact check, do we make server calls to check for features. Because that may lead to transient fault handling in test execution condition. I like the idea of dynamically checking the server for the feature, as long as it is not done in the body of the test, since the test would show as executed but it really wont be executed. So adding a server query in ConditionalFact condition would make sense. But wanted to check if there is prior art.